config/clusters/2i2c-aws-us/cluster.yaml

```diff
@@ -26,14 +26,14 @@ hubs:
       - daskhub-common.values.yaml
       - dask-staging.values.yaml
       - enc-dask-staging.secret.values.yaml
-  - name: researchdelight
-    display_name: "2i2c Research Delight"
-    domain: researchdelight.2i2c.cloud
+  - name: showcase
```

Comment:
This change means that a new hub and helm namespace will be deployed. Hence the need to delete the old `researchdelight` namespace documented in the top comment.